schema/data/prod/product.sql

```diff
@@ -39,7 +39,11 @@ values
 (33, 'Petroleum Refining','BC refining complexity factor unit','active',true),
 (34, 'Wood pellets', 'bone dry tonnes','active', true),
 (35, 'Wood chips', 'bone dry tonnes','active', true),
-(36, 'Waste rendering', 'tonnes','active', true)
+(36, 'Waste rendering', 'tonnes','active', true),
+(37, 'Imported electricity', 'MWh', 'active' ,true),
```

Comment:
```suggestion
(37, 'Purchased electricity', 'GWh', 'active' ,true),
```
Naming and units changes as requested by CAS